docs/elements/hx-icon/index.html

```diff
@@ -1,5 +1,7 @@
 ---
-title: <hx-icon>
+title: Icons
```

Comment:
Shouldn't this title still be `<hx-icon>` since this is the element page?

Reply:
yes good catch thank you -- this is why i love p/r reviews lol

Reply:
yes GREAT CATCH -- why i love p/r reviews